src/main/java/com/javamas/araignee/mq/MessageQueueManager.java

```diff
@@ -0,0 +1,96 @@
+package com.javamas.araignee.mq;
+
+import java.io.IOException;
+
+import com.fasterxml.jackson.core.JsonProcessingException;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import com.rabbitmq.client.Channel;
+import com.rabbitmq.client.Connection;
+import com.rabbitmq.client.ConnectionFactory;
```

Comment:
\[Checkstyle\] WARNING: Import statement for 'com\.rabbitmq\.client\.ConnectionFactory' is in the wrong order\. Should be in the 'THIRD\_PARTY\_PACKAGE' group, expecting not assigned imports on this line\.